docs/pages/admin-guides/infrastructure-as-code/managing-resources/access-list.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Creating Access Lists with IaC
-description: Use Infrastructure-as-Code tooling to create Teleport AccessLists.
+description: Use Infrastructure-as-Code tooling to create Access Lists.
```

Comment:
Same question here as "Teleport Access Requests" and "Teleport Access Controls".